src/Controller/Component/PropertiesComponent.php

```diff
@@ -95,9 +95,11 @@ public function startup(): void
 
             return;
         }
-
+        /** @var \App\Controller\ModulesController $controller */
```

Comment:
we could add the `Config` component to `PropertiesComponent` simply declaring

```php
    protected $components = ['Config'];
```
Then simply call `$this->Config` methods inside the component